google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/BucketInfoTest.java

```diff
@@ -205,4 +212,21 @@ public void testDeleteRules() {
       assertEquals(delRule, DeleteRule.fromPb(delRule.toPb()));
     }
   }
+
+  @Test
+  public void testLifecycleRules() {
+    Rule deleteLifecycleRule = new LifecycleRule(new LifecycleRule.DeleteLifecycleAction(),
+            new LifecycleRule.LifecycleCondition.Builder().setAge(10).build()).toPb();
+
+    assertEquals("Delete", deleteLifecycleRule.getAction().getType());
+    assertEquals(10, (long)deleteLifecycleRule.getCondition().getAge());
+
+    Rule setStorageClassLifecycleRule = new LifecycleRule(
+            new LifecycleRule.SetStorageClassLifecycleAction(StorageClass.COLDLINE),
+            new LifecycleRule.LifecycleCondition.Builder().setIsLive(true).setNumberOfNewerVersions(10).build()).toPb();
+
+    assertEquals("COLDLINE", setStorageClassLifecycleRule.getAction().getStorageClass());
```

Comment:
Use `StorageClass.COLDLINE`, I think the return type here should follow the same type used for `StorageClass.COLDLINE`.